docker-compose.yml

```diff
@@ -1,21 +1,32 @@
-db:
-  image: postgres
-elasticsearch:
-  image: elasticsearch:2
-rabbitmq:
-  image: rabbitmq
-web:
-  image: ccnmtl/locustempus
-  environment:
-    - APP=plexus
-    - SECRET_KEY=dummy-secret-key
-    - SETTINGS=settings_compose
-  command: manage runserver 0.0.0.0:8000
-  volumes:
-    - .:/app/
-  ports:
-    - "8000:8000"
-  links:
-    - db
-    - elasticsearch
-    - rabbitmq
+services:
+  db:
+    image: postgis/postgis:13-3.3
```

Comment:
Similarly, we can pin to the newer version of postgis for development purposes: `postgis/postgis:15-3.3`